ENIGMAsystem/SHELL/Graphics_Systems/OpenGL3/textures.cpp

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this file for?

Reply:
couldnt decide where to put something. forgot to delete